### PR TITLE
component.js import path generation normalized

### PR DESCRIPTION
**Pull request by petergaal91**
This PR fixes the component.js import path generation on Windows machines in `@uxpin/merge-cli` package.

With this fix, the WSL usage not needed anymore.

Issue was the following:
```
ERROR in ./.uxpin-merge/components.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
SyntaxError: C:\Users\peter\workspace\...\.uxpin-merge\components.js: Bad character escape sequence. (4:49)

  2 | import * as ReactDOM from 'react-dom';
  3 | import Button from '..\packages\elements\src\custom-elements\button\button.wrapper';
> 4 | import Wrapper from '..\packages\elements\build\uxpin\uxpin.wrapper.js';
    |                                                  ^
  5 | export {
  6 |   Button,
  7 |   Wrapper,
```

### CODE REVIEW

File: packages/uxpin-merge-cli/src/steps/building/library/getLibraryBundleSource.ts [hunk 1]
```diff
@@ -36,6 +36,10 @@ export function getLibraryBundleSource(components:ComponentDefinition[], wrapper
   ].join('\n');
 }
 
+function normalizePath(path: string):string {
```
Comment (robhil) [inline packages/uxpin-merge-cli/src/steps/building/library/getLibraryBundleSource.ts:39]:
```suggestion
function normalizePath(path:string):string {
```

**Review by robhil — COMMENTED** [review]
Thank you for your contribution. Could you add a test for getLibraryBundleSource which covers your case?

File: packages/uxpin-merge-cli/src/steps/building/library/getLibraryBundleSource.ts (hunk 1 above)
Comment (petergaal91) [reply packages/uxpin-merge-cli/src/steps/building/library/getLibraryBundleSource.ts]: I fixed this, thanks

**Comment by petergaal91** [comment]
Sure! I added one test. Hope circleci build will be good

**Comment by michaelrambeau** [comment]
Closing this PR as the issue was handled in another PR mentioned in the thread.

**Pull request closed without merging**